File: src/java/ards-adif-api/src/main/java/com/ards/adif/api/model/QslSent.java
package com.ards.adif.api.model;

import java.io.Serializable;
import javax.persistence.*;


/**
 * The persistent class for the qsl_sent database table.
 * 
 */
@Entity
@Table(name="qsl_sent")
@NamedQuery(name="QslSent.findAll", query="SELECT q FROM QslSent q")
public class QslSent implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@Column(unique=true, nullable=false)
	private Integer id;

	@Column(length=255)
	private String description;

	@Column(nullable=false, length=20)
	private String meaning;

	@Column(nullable=false, length=1)
	private String status;

	public QslSent() {
	}

	public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getDescription() {
		return this.description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public String getMeaning() {
		return this.meaning;
	}

	public void setMeaning(String meaning) {
		this.meaning = meaning;
	}

	public String getStatus() {
		return this.status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

}

File: src/dotnet/ArdsTools/Ards.Tools.Domain/Models/QslSent.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Ards.Tools.Domain.Adif.Adif.Models
{
    [Table("qsl_sent", Schema = "adif")]
    public partial class QslSent
    {
        [Column("id")]
        public int Id { get; set; }
        [Required]
        [Column("status")]
        [StringLength(1)]
        public string Status { get; set; }
        [Required]
        [Column("meaning")]
        [StringLength(20)]
        public string Meaning { get; set; }
        [Column("description")]
        [StringLength(255)]
        public string Description { get; set; }
    }
}
